examples/ASCollectionView/Sample/SupplementaryNode.m

```diff
@@ -0,0 +1,54 @@
+/* This file provided by Facebook is for non-commercial testing and evaluation
+ * purposes only.  Facebook reserves all rights not expressly granted.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+ * FACEBOOK BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN
+ * ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+#import "SupplementaryNode.h"
+
+#import <AsyncDisplayKit/ASDisplayNode+Subclasses.h>
+#import <AsyncDisplayKit/ASInsetLayoutSpec.h>
+#import <AsyncDisplayKit/ASCenterLayoutSpec.h>
+
+static CGFloat kInsets = 15.0;
+
+@implementation SupplementaryNode {
+  ASTextNode *_textNode;
+}
+
+- (instancetype)initWithText:(NSString *)text
+{
+  self = [super init];
+  if (self != nil) {
+    _textNode = [[ASTextNode alloc] init];
+    _textNode.attributedString = [[NSAttributedString alloc] initWithString:text
+                                                                 attributes:[self textAttributes]];
+    [self addSubnode:_textNode];
+  }
+  return self;
+}
+
+- (ASLayoutSpec *)layoutSpecThatFits:(ASSizeRange)constrainedSize
+{
+  ASCenterLayoutSpec *center = [[ASCenterLayoutSpec alloc] init];
```

Comment:
I'm not actually certain, but is the Center spec of value / does it do anything in this use case where the Inset is also present?